app/views/mortgage_calculator/stamp_duties/_first_time_buyer_table.html.erb

```diff
@@ -0,0 +1,33 @@
+<table class="mortgagecalc__table stamp-duty__ftb-table">
+  <thead>
+    <tr>
+      <th class="mortgagecalc__table__price">
+        <%= I18n.t('stamp_duty.table.property_price_header') %>
+      </th>
+      <th class="mortgagecalc__table__rate">
+        <%= I18n.t('stamp_duty.table.rate_header') %>
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <% ftb_bands.count.times do |i| %>
```

Comment:
why `.count.times` rather than `.each` ?

Reply:
@willhall88 yes we can use either `<% ftb_bands.count.times do |i| %>` or `<% ftb_bands.each_with_index do |_, i| %>`. We just want the _count_, `i`.

Reply:
but why are we needing `i`? you are only using that to select the index in the array.. if you just cycle through it using `.each` then you achieve this same result without having to find it by index?

Reply:
```
<% ftb_bands.each do |band| %>
...
         <%= band(band[:lower], band[:upper]) %>
...
          <%= band[:single_home] %>
...

<% end %>
```